botcore/__init__.py

```diff
@@ -1,11 +1,15 @@
 """Useful utilities and tools for Discord bot development."""
 
-from botcore import exts, site_api, utils
+from botcore import async_stats, exts, site_api, utils
+from botcore._bot import BotBase, StartupError
 
 __all__ = [
+    async_stats,
```

Comment:
I think the only thing worth exporting from this module is `AsyncStatsClient`. If you want to export the whole module then you should add an `__all__` to it.

Reply:
I think exported the module with an `__all__` is better here, otherwise the documentation loses the namespace and gets put directly under botcore. If we start doing that a lot it'll get quite busy.

Will do the same for the comment below

Reply:
https://github.com/python-discord/bot-core/pull/42/commits/7d67c3fd6ebabe9dc92ed379e7705e36cf147c5d